modules/augment/misc/python/test/test_average_blur.py

```diff
@@ -7,10 +7,8 @@
 np.random.seed(seed=1)
 cv.setRNGSeed(seed=1)
 
-transformations = [cv.augment_AverageBlur(minKernelSize=3, maxKernelSize=15),
-                   cv.augment_AverageBlur(minKernelSize=(5,7), maxKernelSize=(13,15)),
-                   cv.augment_AverageBlur(kernelSize=7),
-                   cv.augment_AverageBlur(kernelSize=(5,7))]
+transformations = [cv.augment_AverageBlur(minKernelSize=(5,7), maxKernelSize=(13,15)),
+                   cv.augment_AverageBlur(kernelSize=(5,7), borderType=cv.BORDER_REPLICATE)]
```

Comment:
Please keep tests for all the cases.

Reply:
The constructor(int kernelSize, int borderType = default) was removed because it conflicts with (int minKernelSize, int maxKernelSize , int borderType = default)

the constructor(Size kernelSize, borderType) is sufficient for the same purpose

Reply:
This way we need rather resolve a conflict:

```python
cv.augment_AverageBlur(kernelSize=3)
cv.augment_AverageBlur(minKernelSize=3, minKernelSize=5)
```
are the most important methods. Let's keep the following constructors:

```cpp
AverageBlur(int kernelSize)
AverageBlur(int minKernelSize, int maxKernelSize, int borderType = BORDER_DEFAULT)
```

Reply:
done